CONTRIBUTING.md

```diff
@@ -502,6 +522,59 @@ $ popd
 >**Note:** The final `popd` is required to return to the top-level directory 
 >from where other build rules can be executed.
 
+## Porting
+
+Porting is the action of applying a patch set to either an older (backwards
+port, or simply "backport") or newer (forward port) branch or repository.
```

Comment:
Consider a revision to simplify wording: 
Porting applies a patch set to an older ("backport") or a newer ("forward-port") branch or repository.
[...] You must add a porting label. Porting PRs *must* have an associated "parent" PR....